packages/hash/frontend/jest.config.js

```diff
@@ -13,4 +13,14 @@ const customJestConfig = {
   testEnvironment: "jest-environment-jsdom",
 };
 
-module.exports = createJestConfig(customJestConfig);
+module.exports = async () => {
+  const nextJestConfig = await createJestConfig(customJestConfig)();
+  // https://github.com/vercel/next.js/issues/36230
+  nextJestConfig.moduleNameMapper = {
+    // Workaround to put our SVG stub first
+    "\\.svg": "<rootDir>/src/__mocks__/file.mock.ts",
```

Comment:
Needed because of a breaking change in `next/jest`. Not adding `moduleNameMapper` for svgs breaks support for this part of `next.config.js`:

```
        webpackConfig.module.rules.push({
          test: /\.svg$/,
          use: [require.resolve("@svgr/webpack")],
        });
```

`import { LogoIcon } from "../../shared/icons"` is an object instead of JSX, so `<LogoIcon />` crashes React:

```
React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.
```

https://github.com/vercel/next.js/issues/36230